subsys/net/lib/nrf_cloud/src/nrf_cloud_codec_internal.c

```diff
@@ -968,8 +969,46 @@ int json_send_to_cloud(cJSON *const request)
 
 	return err;
 }
+
+static int shadow_connection_reset(cJSON *connection_obj)
```

Comment:
After a couple re-reads I see now that `shadow_connection_reset` is meant to write appropriate JSON data so-as to cause connection info to be reset. So it's "reset the shadow connection info" -- but the title reads a bit more like "reset the shadow connection"

Maybe it could be named something like `shadow_connection_info_blank`.

Either way it would be good to have some function briefs and a couple more inline comments for `shadow_connection_reset` and `shadow_connection_update`

Reply:
The verb needs to go on the right.  These are static functions which we normally do not provide briefs on.